File: C Sharp/C# Advanced/5-Strings/Task3_CorrectBrackets/CorrectBrackets.cs
using System;

namespace Task3_CorrectBrackets
{
    class CorrectBrackets
    {
        static void Main(string[] args)
        {
        string text = Console.ReadLine();
        string result = CounterBrackets(text);
        Console.WriteLine(result);
        }
    static string CounterBrackets(string str)
    {
        int counter = 0;
        for (int i = 0; i < str.Length; i++)
        {
            if (str[i] == '(')
            {
                counter++;
            }
            else if (str[i] == ')')
            {
                counter--;
            }
        }
        if (counter == 0)
        {
            return "Correct";
        }
        else
        {
            return "Incorrect";
        }
    }
}
}

File: Front-end/JavaScript-Fundamentals/07. Strings/02. Correct brackets/correctBrackets.js
function solve(args) {
    let countOpen = 0,
        countClose = 0,
        text = args[0];

    for (let i = 0; i < text.length; i += 1) {
        if (text[i] === '(') {
            countOpen += 1;
        }
        if (text[i] === ')') {
            countClose += 1;
        }
    }
    if (countOpen === countClose) {
        console.log('Correct');
    } else {
        console.log('Incorrect');
    }


}

//solve(['((a+b)/5-d)']);
//solve([')(a+b))']);